.github/workflows/deploy-gh-pages.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: npm run build
 
       - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+        uses: actions/upload-artifact@v4
```

Comment:
I don't think this works for Pages deployment. From what I remember you need to use

```
actions/upload-pages-artifact@v3
actions/deploy-pages@v4
```

and update them together.

See
- https://github.com/EnAccess/micropowermanager/blob/main/.github/workflows/deploy-gh-pages.yaml#L47
- https://github.com/EnAccess/micropowermanager/blob/main/.github/workflows/deploy-gh-pages.yaml#L61

Reply:
Oh I see, Thanks for pointing out 🙌🏾

Reply:
Updated.